day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,27 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new_topping)
+    toppings.push(new_topping)
+    puts "I dont see #{new_topping} on your menu but can you please add some #{new_topping} for me?!"
+  end
+
+  def remove_topping
+    puts "Please remove the #{toppings[0]}, many thanks!"
+  end
```

Comment:
How would you actually remove a topping from the toppings array with code here? 

For reference, your `add_topping` method on line 16 adds a new topping to the toppings array with `toppings.push(new_topping)`.